systems/WikibaseClient/05-Building_Block_View.md

```diff
@@ -65,15 +65,16 @@
 
 ## Client Side Item Edits
 
-## Linked Site Page Changes
+## Sitelinks
```

Comment:
I actually wanted to call this header "Interwiki / Sitelinks" but couldn't make the internal anchor links work without complaints from the linter.

Reply:
I am uncomfortable with this title change.

"Sitelinks" are not a concept that really exists in the Client domain
The `UpdateRepo` jobs do not operate on or with sitelinks or interwiki links.

Further, the populateInterwiki maint script does not do anything with "sitelinks" either.
It infact operates alone and only interacts with a single MediaWiki database table (not technically a client DB table)
It may make sense to separate the client DB and the client mediawiki DB into separate blocks?

I think this populateInterwiki script probably lives in an interwiki block that we simply havn't created yet.
I imagine more of the content will likely come from the hooks that we have not looked at.

Thinking further forward, we also have this extension https://www.mediawiki.org/wiki/Extension:InterwikiSorting which was extracted from client, but is deployed with all clients, that probably also lives in this area?